### PR TITLE
Disable hot event listeners fixes #4861

### LINKED ISSUE #4861: Disable event listeners with max points
We currently allow users to view console logs for event listeners with more than the max points. We should disable that in the same way that we disable breakpoints with more than the max number of hits.

Screen Shot 2021-12-17 at 8 07 49 PM

### PR DESCRIPTION
**Pull request by jasonLaster**

### CODE REVIEW

File: src/devtools/client/debugger/src/components/SecondaryPanes/EventListeners.js [hunk 1]
```diff
@@ -295,16 +295,21 @@ class EventListeners extends Component {
       return null;
     }
 
+    const isHot = points.length > prefs.maxHitsEditable;
+    const title = isHot ? `Cannot view ${event.name} events` : `View ${event.name} events`;
+    console.log(isHot, points.length);
```
Comment (jcmorrow) [inline src/devtools/client/debugger/src/components/SecondaryPanes/EventListeners.js:300]: ✂️

File: src/devtools/client/debugger/src/components/SecondaryPanes/EventListeners.js [hunk 2]
```diff
@@ -295,16 +295,21 @@ class EventListeners extends Component {
...
+
     return (
       <li
-        className="px-2 pl-5 mb-1 mr-1 flex flex-row items-center hover:bg-gray-200 rounded-md"
+        className="px-2 pl-5 mb-1 mr-1 flex flex-row items-center hover:bg-gray-100 rounded-md"
         key={event.id}
       >
         <div className="flex flex-row justify-between w-full">
-          <label className="w-full flex flex-row items-center">
+          <label title={title} className="w-full flex flex-row items-center">
```
Comment (jcmorrow) [inline src/devtools/client/debugger/src/components/SecondaryPanes/EventListeners.js:308]: Is this just for the hover text?

File: src/devtools/client/debugger/src/components/SecondaryPanes/EventListeners.js [hunk 3]
```diff
@@ -313,11 +318,9 @@ class EventListeners extends Component {
               {searchText ? this.renderCategory(category) : null}
               {event.name}
             </span>
-            {features.eventCount ? (
```
Comment (jcmorrow) [inline src/devtools/client/debugger/src/components/SecondaryPanes/EventListeners.js:316]: what is eventCount supposed to be doing here? It looks like now in the case where `features.eventCount` is false but there are `points` we will still render all of this, is that right?

**Review by jcmorrow — COMMENTED** [review]
Just a couple questions and a `console.log`

File: src/devtools/client/debugger/src/components/SecondaryPanes/EventListeners.js (hunk 3 above)
Comment (jasonLaster) [reply src/devtools/client/debugger/src/components/SecondaryPanes/EventListeners.js:316]: yeah - i think it's safe to remove this

File: src/devtools/client/debugger/src/components/SecondaryPanes/EventListeners.js (hunk 2 above)
Comment (jasonLaster) [reply src/devtools/client/debugger/src/components/SecondaryPanes/EventListeners.js:308]: yep

**Pull request merged**